<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>main_window</class>
    <widget class="QMainWindow" name="main_window">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>952</width>
                <height>686</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>952</width>
                <height>686</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Swiftee</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="central_widget_layout">
                <property name="spacing">
                    <number>0</number>
                </property>
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <item>
                    <widget class="QToolBar" name="main_toolbar">
                        <property name="minimumSize">
                            <size>
                                <width>0</width>
                                <height>38</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>38</height>
                            </size>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QSplitter" name="main_splitter">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="handleWidth">
                            <number>0</number>
                        </property>
                        <widget class="QWidget" name="navigator_layout_widget">
                            <layout class="QVBoxLayout" name="navigator_layout">
                                <property name="spacing">
                                    <number>0</number>
                                </property>
                                <property name="sizeConstraint">
                                    <enum>QLayout::SetDefaultConstraint</enum>
                                </property>
                                <item>
                                    <widget class="QToolBar" name="navigator_switch">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>28</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>16777215</width>
                                                <height>28</height>
                                            </size>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">QToolBar { background: palette(window); border-color: palette(mid); border-width: 0px 0px 1px 0px; border-style: solid; }</string>
                                        </property>
                                        <property name="movable">
                                            <bool>false</bool>
                                        </property>
                                        <property name="allowedAreas">
                                            <set>Qt::TopToolBarArea</set>
                                        </property>
                                        <property name="floatable">
                                            <bool>false</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QWidget" name="navigator_stack" native="true">
                                        <property name="minimumSize">
                                            <size>
                                                <width>200</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                        <widget class="QSplitter" name="middle_splitter">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="handleWidth">
                                <number>0</number>
                            </property>
                            <widget class="QWidget" name="code_edit_layout_widget">
                                <layout class="QVBoxLayout" name="code_edit_layout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QToolBar" name="code_edit_toolbar">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>9</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">QToolBar { background: white; border-color: palette(mid); border-width: 0px 0px 1px 0px; border-style: solid; }</string>
                                            </property>
                                            <property name="movable">
                                                <bool>false</bool>
                                            </property>
                                            <property name="allowedAreas">
                                                <set>Qt::TopToolBarArea</set>
                                            </property>
                                            <property name="floatable">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="CodeEditView" name="code_edit_view">
                                            <property name="styleSheet">
                                                <string notr="true">border: none;</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                            <widget class="QWidget" name="debug_layout_widget">
                                <layout class="QVBoxLayout" name="debug_layout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QToolBar" name="debug_toolbar">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">QToolBar { background: palette(window); border-color: palette(mid); border-width: 1px 0px 1px 0px; border-style: solid; }</string>
                                            </property>
                                            <property name="movable">
                                                <bool>false</bool>
                                            </property>
                                            <property name="allowedAreas">
                                                <set>Qt::TopToolBarArea</set>
                                            </property>
                                            <property name="floatable">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QSplitter" name="debug_splitter">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="handleWidth">
                                                <number>0</number>
                                            </property>
                                            <widget class="QWidget" name="variables_layout_widget">
                                                <layout class="QVBoxLayout" name="variables_layout">
                                                    <property name="spacing">
                                                        <number>0</number>
                                                    </property>
                                                    <item>
                                                        <widget class="QTreeView" name="variables_view">
                                                            <property name="styleSheet">
                                                                <string notr="true">border: none; selection-color: white;</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QToolBar" name="variables_toolbar">
                                                            <property name="minimumSize">
                                                                <size>
                                                                    <width>0</width>
                                                                    <height>28</height>
                                                                </size>
                                                            </property>
                                                            <property name="maximumSize">
                                                                <size>
                                                                    <width>16777215</width>
                                                                    <height>28</height>
                                                                </size>
                                                            </property>
                                                            <property name="styleSheet">
                                                                <string notr="true">QToolBar { background: white; border-color: palette(mid); border-width: 1px 0px 0px 0px; border-style: solid; }</string>
                                                            </property>
                                                            <property name="movable">
                                                                <bool>false</bool>
                                                            </property>
                                                            <property name="allowedAreas">
                                                                <set>Qt::BottomToolBarArea</set>
                                                            </property>
                                                            <property name="floatable">
                                                                <bool>false</bool>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                            <widget class="QWidget" name="console_layout_widget">
                                                <layout class="QVBoxLayout" name="console_layout">
                                                    <property name="spacing">
                                                        <number>0</number>
                                                    </property>
                                                    <item>
                                                        <widget class="QTextEdit" name="console_view">
                                                            <property name="styleSheet">
                                                                <string notr="true">border: none;</string>
                                                            </property>
                                                            <property name="textInteractionFlags">
                                                                <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QToolBar" name="console_toolbar">
                                                            <property name="minimumSize">
                                                                <size>
                                                                    <width>0</width>
                                                                    <height>28</height>
                                                                </size>
                                                            </property>
                                                            <property name="maximumSize">
                                                                <size>
                                                                    <width>16777215</width>
                                                                    <height>28</height>
                                                                </size>
                                                            </property>
                                                            <property name="styleSheet">
                                                                <string notr="true">QToolBar { background: white; border-color: palette(mid); border-width: 1px 0px 0px 0px; border-style: solid; }</string>
                                                            </property>
                                                            <property name="movable">
                                                                <bool>false</bool>
                                                            </property>
                                                            <property name="allowedAreas">
                                                                <set>Qt::BottomToolBarArea</set>
                                                            </property>
                                                            <property name="floatable">
                                                                <bool>false</bool>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </widget>
                        <widget class="QSplitter" name="utilities_splitter">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="handleWidth">
                                <number>0</number>
                            </property>
                            <widget class="QWidget" name="inspector_layout_widget">
                                <layout class="QVBoxLayout" name="inspector_layout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QToolBar" name="inspector_switch">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">QToolBar { background: palette(window); border-color: palette(mid); border-width: 0px 0px 1px 0px; border-style: solid; }</string>
                                            </property>
                                            <property name="movable">
                                                <bool>false</bool>
                                            </property>
                                            <property name="allowedAreas">
                                                <set>Qt::TopToolBarArea</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QWidget" name="inspector_stack" native="true">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>200</width>
                                                    <height>30</height>
                                                </size>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                            <widget class="QWidget" name="library_layout_widget">
                                <layout class="QVBoxLayout" name="library_layout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QToolBar" name="library_switch">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>28</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">QToolBar { background: white; border-color: palette(mid); border-width: 1px 0px 1px 0px; border-style: solid; }</string>
                                            </property>
                                            <property name="movable">
                                                <bool>false</bool>
                                            </property>
                                            <property name="allowedAreas">
                                                <set>Qt::TopToolBarArea</set>
                                            </property>
                                            <property name="floatable">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QWidget" name="library_stack" native="true">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>190</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">background-color: white</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>952</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menu_file">
                <property name="title">
                    <string>File</string>
                </property>
                <widget class="QMenu" name="menu_new">
                    <property name="title">
                        <string>New</string>
                    </property>
                    <addaction name="action_project"/>
                </widget>
                <addaction name="menu_new"/>
            </widget>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>Edit</string>
                </property>
                <addaction name="actionUndo"/>
                <addaction name="actionRedo"/>
                <addaction name="separator"/>
                <addaction name="actionCopy"/>
            </widget>
            <addaction name="menu_file"/>
            <addaction name="menuEdit"/>
        </widget>
        <action name="action_project">
            <property name="text">
                <string>Project</string>
            </property>
        </action>
        <action name="actionUndo">
            <property name="text">
                <string>Undo</string>
            </property>
        </action>
        <action name="actionRedo">
            <property name="text">
                <string>Redo</string>
            </property>
        </action>
        <action name="actionCopy">
            <property name="text">
                <string>Cut</string>
            </property>
        </action>
    </widget>
    <customwidgets>
        <customwidget>
            <class>CodeEditView</class>
            <extends>QTextEdit</extends>
            <header>sources.codeedit.codeeditview</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
